### ISSUE TITLE
The content of "Extension details" panel is truncating on applying reflow. : A11y_SQL Azure Data Studio_Extension_Reflow

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26396]
### Tags:
#A11yTCS;#A11yWCAG2.1;#WCAG1.4.10;#Reflow;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Pre-Requisites:

1. Adjust the width of entire azure data studio application width to 320pixel.

### Repro Steps:

1.Apply Reflow.
2.  Launch Azure Data Studio.
3. From the left menu, click the 'Extensions' icon to open the Extension Manager.
4. Choose any extension from the library.
5. The Extension details panel will appear.
6.  Observe that The content of "Extension details" panel is truncating on applying reflow.
### Actual:
The content of "Extension details" panel is truncating on applying reflow.

### Expected:
All the content of "Extension Details" panel should be clearly and completely visible on applying reflow.

### User Impact:
Low vision users will not be able to access the complete content of details panel as in reflow view the panel data is truncated.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 1.4.10: Reflow Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-1.4.10--Reflow-Guide.aspx)
- [MAS 1.4.10 – Reflow](https://aka.ms/MAS1.4.10)

<img width="1365" height="730" alt="Image" src="https://github.com/user-attachments/assets/7949f68d-62b3-42a0-9ac2-edcf5f781300" />
